lib/gooddata/lcm/types/scalar/string.rb

```diff
@@ -16,6 +16,15 @@ class StringType < BaseType
         def check(value)
           value.is_a?(String)
         end
+
+        def migrate(value, to_type)
+          case to_type
+          when GoodData::LCM2::Type::ArrayType
+            value.nil? ? (true) : ([value])
```

Comment:
Refers to 'value' more than self (maybe move it to another class?) (FeatureEnvy)